Documentation/Signoffs/SensorPosts.md

```diff
@@ -0,0 +1,133 @@
+# Sensor Post Array Velocity and Acceleration Signoff
+## Functionality 
+ This portion of the sensor post array will be responsible for obtaining the velocity and acceleration of the practice golf ball as it travels down the fishing line towards the interceptor body. 
+ To accomplish this task this subsystem will take in the projectiles location data from a mmWave Radar unit.
+ This unit will detect the presense of an object and will do time of flight calculations to obtain the velocity and acceleration.
+ This subsystem will then wirelessly transmit the values to the aiming subsystem for further use of this data.
+
+## Constraints
+<div align="center">
+
+|Constraint Number | Description | Origin |
+|------------------|-------------|--------|
+| 1 | The design shall be autonomous, and will require no outside interference after initial startup. | The customer requires the interceptor and all external sensor posts must be autonomous after initial startup. |
+| 2 | The design shall operate on battery power for a minimum of 30 minutes. | This comes from the requirements set forth in the rulebook by devcom, and is further constrained based on the power consumption of this subsystem.|
+| 3 | The design shall have maximum sensor input delay of 100ms. | This comes from a timing contraint for intercepting the projectile. |
+| 4 | The Design shall have an alternative power method to be used during testing. | This is a requirement for lessening enviornmental impact of our interceptor. |
+| 5 | The design shall have a mimumum range of 45 inches from the point of placement. | This constraint arises from the range of detection for the projectile and time to obtain appropriate data within the time window. |
+| 6 | This design shall have a placement to put the radar beam angle within 100*40 $^{\circ}$ . | This constraint is derived from the beam angle specified by the mmWave Radar manufacturer |
```

Comment:
This makes sense. But how do you know that this will cover the field appropriately? Show your math in the discussion below